routemaster/feeds.py

```diff
@@ -28,6 +28,15 @@ def __init__(self, url, state_machine):
         self.data = None
         self.state_machine = state_machine
 
+    def __eq__(self, other):
+        if not isinstance(other, Feed):
+            return False
+        return (
+            self.url == other.url and
+            self.state_machine == other.state_machine and
+            self.data == other.data
```

Comment:
I'm not sure it's a good idea to base equality on mutable state.

Related: should we also be overriding `__hash__`?

Reply:
Agreed. This might be a bit much, but I planned on moving in this direction anyway. 20504bddc433244a62fe2f1a4d826e3727a75545